.changelog/2306.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/mongodbatlas_network_peering: Correctly handle GCP updates of mongodbatlas_network_peering
```

Comment:
```suggestion
resource/mongodbatlas_network_peering: Correctly handles GCP updates of mongodbatlas_network_peering
```

Reply:
knit: as we're already specifying the resource in the beginning, we might just say: 
resource/mongodbatlas_network_peering: Correctly handles GCP updates